### ISSUE TITLE
partial rsync error when parsing zip file from HT rsync to index page data

### ISSUE THREAD

**Issue by mnaydan** [Princeton-CDH/ppa-django#539]
Discovered when trying to add uga1.32108002998303 to the database. 

Culprit from logs:
> KeyError: "There is no item named '$b31619/UCAL_$B31619_00000177.txt' in the archive"

It seems accurate that the text file is not in the zipfile but we're not sure why the code expects it to be. Looking at the mets xml, the pages only go up to 108.txt.

**Comment by rlskoeser** [comment]
The record that's actually causing this problem is uc1.$b31619

I got an error when I tried to suppress it in QA - we don't have permission (at least in qa) to delete the pairtree files from the NFS mount point. That's something I'll have to check on with Francis, but we may want an issue to track it.  (Would be nice to know if it's also a problem in production)

**Comment by mnaydan** [comment]
How would we check whether that's a problem in production without actually suppressing it? I suppose I could suppress, delete, and then re-add it if you think I should. The record appears to be working and indexed in production: [https://prosody.princeton.edu/archive/uc1.$b31619/?query=epic](https://prosody.princeton.edu/archive/uc1.$b31619/?query=epic)

**Comment by rlskoeser** [comment]
Yeah, that was my concern too - would rather not delete or suppress something we don't want to actually delete.  I'll try to figure it out from the filesystem side without needing that.

It's possible that's _not_ the record that's actually causing the problem... so weird!

**Comment by mnaydan** [comment]
Looking at it on HathiTrust, it looks like page 176 is the last meaningful page (and so the last one with a txt file); the subsequent pages are blank, flyleaf, and back cover, so we are not missing content. That plan sounds good to me -- to add error handling so it's not fatal, and log. To confirm: Is trying to call txt files for flyleafs, covers, etc. not a problem for other works? 

I can submit a "Content Quality Correction" for the work on HathiTrust but that seems to be for scan quality on the frontend.
